tic-tac-toe/src/components/game/game.js

```diff
@@ -0,0 +1,127 @@
+import React, {Component} from 'react';
+import Board from '../board/board';
+import ControlPannel from '../control-pannel/control-pannel';
+import GameInfo from '../info-bar/info-bar';
+import InfoBlock from '../info-block/info-block';
+import checkTurn from '../../scripts/calculate-next-move';
+import calculateWinner from '../../scripts/calculate-winner';
+import {getGameData, updateGameData} from '../../scripts/storage-controller';
+import {gameModes} from '../../libraries/library';
+import classes from './game.module.css';
+
+class Game extends Component {
+    constructor(props) {
+    super(props);
+    this.isBlockGame = false;
```

Comment:
Could you elaborate this name a little ? Did you mean `isGameBlocked` ? Anyway boolean name should be like a questions with true or false answer

Reply:
I renamed this variable. I hope the name describes the meaning.
```suggestion
    this.isNextTurnBlocked = false;
```